src/runtime/runtime_rp2040.go

```diff
@@ -43,6 +50,284 @@ func sleepTicks(d timeUnit) {
 	}
 }
 
+// Currently sleeping core, or 0xff.
```

Comment:
```sh
$ diff src/runtime/runtime_rp2*
1c1
< //go:build rp2040
---
> //go:build rp2350
31a32,35
>       if d <= 0 {
>               return
>       }
>
```

Can the two files be merged, unlocking multicore on rp2350 for (approximately) free?

Reply:
I will give you one next week!